### PR TITLE
GatherND_8 specification

### PR DESCRIPTION
**Pull request by achetver**
### Details:
 - Add specification for GatherND_8 operation.

### Tickets:
 - 66646

### CODE REVIEW

File: docs/ops/movement/GatherND_5.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ and *GatherND* starts to gather from the `b+1` dimension.
 It requires the first `b` dimensions in `data` and `indices` tensors to be equal.
 In case non default value for *batch_dims* the output shape is calculated as
 `(multiplication of indices.shape[:b]) + indices.shape[b:-1] + data.shape[(indices.shape[-1] + b):]`.
+    **NOTE: the calculation of output shape is incorrect for non-default *batch_dims* value**
```
Comment (andrei-kochin) [inline docs/ops/movement/GatherND_5.md:60]: Please refer to GatherND-8 from here

Comment (achetver) [reply docs/ops/movement/GatherND_5.md]: Done.

File: docs/ops/movement/GatherND_8.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+from `data` tensor. A position `i_0, ..., i_{K-2}` in the `indices` tensor corresponds to a tuple with indices `indices[i_0, ..., i_{K-2}]`
+of a length equal to `indices.shape[-1]`. By this tuple with indices the operation gathers a slice or an element from `data` tensor and
+insert it into the output at position `i_0, ..., i_{K-2}` as the following formula:
+
+`output[i_0, ..., i_{K-2},:,...,:] = data[indices[i_0, ..., i_{K-2}],:,...,:]`
+
+The last dimension of `indices` tensor must be not greater than a rank of `data` tensor, i.e. `indices.shape[-1] <= data.rank`.
+
+The shape of the output is calculated as `indices.shape[:batch_dims] + indices.shape[batch_dims:-1]`
+if `indices.shape[-1] == data.rank - batch_dims`  else `indices.shape[:batch_dims] + list(indices.shape)[batch_dims:-1] + list(data.shape)[batch_dims + indices.shape[-1]:]`.
```
Comment (popovaan) [inline docs/ops/movement/GatherND_8.md:21]: Please, split this line into multiple with no more than 120 characters.

**Comment by achetver** [comment]
@andrew-zaytsev @tsavina  Please, review this specification from doc maintainers side.

**Comment by rkazants** [comment]
@jane-intel, could you please take a look and merge if it is fine?

**Pull request merged**